ops/framework.py

```diff
@@ -112,16 +112,16 @@ def restore(self, snapshot):
         self.deferred = False
 
 
-class Event:
-    """Event creates class descriptors to operate with events.
+class EventSource:
+    """EventSource creates class descriptors to operate with events.
```

Comment:
I wonder if we tweak this doc string if it also would help make it clearer why you want to use it.